exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTest.java

```diff
@@ -361,12 +407,14 @@ void createBlockWithTransactionsVarargs() {
     }
   }
 
-  private TransactionMessage constructTestTransactionMessage(String payload) {
+  private TransactionMessage constructTestTransactionMessage(String payload, TestKit testKit) {
+    EmulatedNode emulatedNode = testKit.getEmulatedNode();
+    KeyPair emulatedNodeKeyPair = emulatedNode.getServiceKeyPair();
     return TransactionMessage.builder()
         .serviceId(TestService.SERVICE_ID)
         .transactionId(TestTransaction.ID)
         .payload(payload.getBytes(BODY_CHARSET))
-        .sign(KEY_PAIR, CRYPTO_FUNCTION);
+        .sign(emulatedNodeKeyPair, CRYPTO_FUNCTION);
```

Comment:
I think it is not appropriate to abuse this method in tests that do not _require_ transactions submitted by the service. Such transactions can be signed by any key, and passing testkit and extracting the key of the emulated node is irrelevant to these tests.

I'd make a couple of overloads:
```
constructTestTransactionMessage(payload)
constructTestTransactionMessage(payload, keypair|emulatedNode)
```